en/pp_limiter_en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Datenshi Performance Points Limiter"
+old_id: 13
+---
+<div style="text-align:center"><img src="https://user-images.githubusercontent.com/10250068/116959299-c8ae3580-accf-11eb-9ca3-15cd662dbca0.png" /></div>
+
+Hello! this is guide for who got messages from our Bot In-game, we are introduce to you that we have a unique pp limiter, this limiter are mostly prevent abusers/cheater to have more pp, **please note that you are not in banned/restricted mode**. It's just that you won't get more pp, if you play another fresh beatmap you will be get 0pp.
```

Comment:
```suggestion
Hello! this is a guide for those who got a message from our Bot In-game. We introduce you that we have a unique pp limiter, this limiter is mostly to prevent an abuser/cheater to have more pp, **please note that you are not in banned/restricted mode**. It's just that you won't get more pp, if you play another fresh beatmap you will only get 0pp.
```
grammar fix